pkg/config/config.go

```diff
@@ -1236,6 +1237,14 @@ func GetDogstatsdMappingProfiles() ([]MappingProfile, error) {
 
 func getDogstatsdMappingProfilesConfig(config Config) ([]MappingProfile, error) {
 	var mappings []MappingProfile
+	if v := os.Getenv("DD_DOGSTATSD_MAPPER_PROFILES"); v != "" {
```

Comment:
There is no need to explicitly read env var directly, viper is initialised here https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config.go#L133 and declare the prefix `DD` so env var like `DD_PARAMETER=xyz` are automatically accessible through the config key `parameter`. For a similar use case to the one you are working on you can check https://github.com/DataDog/datadog-agent/blob/master/pkg/logs/config/config.go#L80-L91 that handle complex config extraction from both env var and config file. There should be a generic helper inside our config package to handle that on a general basis, but right now there is not.

Reply:
➕ , but note that since #6347 there is a helper that allows setting a function that defines how the value is extracted from an env var (`SetEnvKeyTransformer`). It's not very generic for now, but I think it makes sense to try to use it for the purposes of this PR.